### PR TITLE
Add Homepage design

### LINKED ISSUE #138: Apply homepage design

### PR DESCRIPTION
**Pull request by slorenzo**
https://user-images.githubusercontent.com/15804684/147808866-56cd387a-1a3f-493c-ba52-ece76b3dc64f.mov

Closes #138

### CODE REVIEW

File: packages/dapp/components/DonationSlide/DonationSlide.js [hunk 1]
```diff
@@ -1,30 +1,47 @@
...
-        min={INITIAL_DONATION - 1}
+        min={MIN_DONATION}
         max={MAX_DONATION}
         defaultValue={INITIAL_DONATION}
         onChange={setDonation}
         step={0.1}
       />
+      <div className={styles.range}>
+        <span>0.2 ETH</span>
+        <span>14 ETH</span>
```
Comment (mariobyn) [inline packages/dapp/components/DonationSlide/DonationSlide.js:37]: Do we actually need a max donation? Can we put like 100 ETH?

File: packages/hardhat/hardhat.config.js [hunk 2]
```diff
@@ -25,6 +25,9 @@ module.exports = {
   networks: {
     local: {
       url: 'http://localhost:8545',
+      accounts: {
+        mnemonic: 'test test test test test test test test test test test junk',
```
Comment (mariobyn) [inline packages/hardhat/hardhat.config.js:29]: do we need this?

Comment (slorenzo) [reply packages/hardhat/hardhat.config.js:29]: It's not required but useful. I always need to open the hardhat wallet and need to go the docs. It's an easier way to found it.

File: packages/dapp/components/DonationSlide/DonationSlide.js (hunk 1 above)
Comment (slorenzo) [reply packages/dapp/components/DonationSlide/DonationSlide.js:37]: It's on the proposal, let's talk with the team.

**Review by mjlescano — APPROVED** [review]

**Pull request merged**